src/storage_utils.cpp

```diff
@@ -357,7 +357,7 @@ namespace libtorrent { namespace aux
 			// need to delete the source.
 			if (copied_files[i] == false) continue;
 
-			std::string cur_file_path{ f.file_path(i) };
+			std::string const cur_file_path{ f.file_path(i) };
```

Comment:
``cur_file_path`` is only used once here. I think capturing this in a named variable just adds noise. It makes sense to capture above because the variable is used twice (and the work done by ``file_path(i)`` only has to be done once).

I would suggest:
```std::string const old_path = combine_path(save_path, make_sv(f.file_path(i)));
```

does that make sense?